### DISCUSSION TITLE
Raspberry Pi Pico W

### DISCUSSION THREAD

**Discussion by jardous** [arendst/Tasmota#20147] [Q&A] [answered]
Any plans to add support for Raspberry Pi Pico W?

**Comment by barbudor** [comment, marked as answer]
No really
Tasmota is quite intrinsically linked to ESP, porting to anything else would be a major time investment for what gain?
Just because RPiPiW exist is not a good reason as there is no commercial devices using that configuration. 

If it was possible, Tuya chips would be much more in Tasmota line of focus.

**Reply by jardous** [reply]
yes this makes total sense...
I just like Tasmota features and would love to have the same for custom devices I am building around the house.
Thanks

**Reply by barbudor** [reply]
Isn't there enough options with ESP based boards?
Why getting RPiPiW? This assembly doesn't makes much sense.

**Reply by sfromis** [reply]
I'd say that what can make the Pico W great for some purposes is the software ecosystem around it. Ditching that, you'd be left with a "meh" board without strong sides. Neither good performance nor compact size.
